manifests/init.pp

```diff
@@ -46,6 +46,8 @@
 #                     if it's not array, it's automatically converted to it, so you can
 #                     even specify just one policy as a hash
 #                     type:array
+#                     
+# $ruby_foreman_scap_client_package:: os dependent package name for rubygem-foreman_scap_client package
```

Comment:
This implies it's a class parameter, but it's not so you either can remove this or add it as a parameter.

Reply:
removed.